src/lib/services-manager/service_loader.rb

```diff
@@ -156,7 +155,6 @@ def extract_services_from_unit_files
           services[name][:loaded] = @supported_units[name][:status] == Status::LOADED
           services[name][:description] = @supported_units[name][:description]
         end
-        services[name][:can_be_enabled] = status == Status::STATIC ? false : true
```

Comment:
Please just be sure that we never try to call enable/disable on it as it will failed.